src/iiif/views.py

```diff
@@ -30,17 +30,17 @@ def index(request, iiif_url):
 
 # TODO: limit to dataportaal urls
 @csrf_exempt
-@ratelimit(key='ip', rate='3/d')  # TODO: Check django cache settings for rate limiter to work across paralel docker containers
+@ratelimit(key='ip', rate='3/d')  # TODO: Check django cache settings for rate limiter to work across parallel docker containers
```

Comment:
For this to work you need to be absolutely sure the IP address is correctly passed to the application. 

> Security considerations
> Client IP address
> 
> IP address is an extremely common rate limit key, so it is important to configure correctly, especially in the equally-common case where Django is behind a load balancer or other reverse proxy.

See https://django-ratelimit.readthedocs.io/en/stable/security.html#client-ip-address

Reply:
Also, will this be called by a user, or internally by dataportaal? In case of the latter, IP address will always be the same, effectively limiting the request amount for all users combined to 3 a day.

Reply:
The dataportaal is only a frontend, so this will have the ip of the user.

Good point on whether the ip is available. I'll check that out.

Reply:
X-Forwarded-For lijkt leeg in Sentry. Ik heb het basis gevraagd.